frontend/app/routes/$lang/_protected/applications/$id.tsx

```diff
@@ -83,18 +83,63 @@ export async function loader({ context: { session }, params, request }: LoaderFu
 
   const allFederalSocialPrograms = lookupService.getAllFederalSocialPrograms();
   const allProvincialTerritorialSocialPrograms = lookupService.getAllProvincialTerritorialSocialPrograms();
+
+  const children = applicationDetails.children?.map((child) => {
+    const selectedFederalBenefit = child.dentalBenefits?.federalSocialProgram && localizeFederalSocialProgram(lookupService.getFederalSocialProgramById(child.dentalBenefits.federalSocialProgram), locale);
+    const selectedProvincialBenefit =
+      child.dentalBenefits?.provincialTerritorialSocialProgram && localizeProvincialTerritorialSocialProgram(lookupService.getProvincialTerritorialSocialProgramById(child.dentalBenefits.provincialTerritorialSocialProgram), locale);
+    console.log(' ');
+    console.log(' ');
+    console.log(' ');
+    console.log(' ');
+    console.log(selectedFederalBenefit);
+    console.log(selectedProvincialBenefit);
+    console.log(' ');
+    console.log(' ');
+    console.log(' ');
+    console.log(' ');
```

Comment:
```suggestion
```

Reply:
Oops. I meant to remove that in my last commit amend. Fixed: 3d1373ce2a5ac1e917845dd099a86ed230dd642a